src/main/java/com/cloudera/sparkts/api/java/DateTimeIndexFactory.java

```diff
@@ -110,6 +110,20 @@ public static IrregularDateTimeIndex irregular(long[] dts, ZoneId zone) {
     }
 
     /**
+     * Create a HybridDateTimeIndex composed of the given indices using the default date-time zone.
```

Comment:
nice catch ... a single instantiator, with a warning in a comment should be sufficient
btw, in a further commit, I've added a helper method, ```atZone```, that returns an index with an adjusted zone.